src/app/utils/ScoapUtils.ts

```diff
@@ -0,0 +1,76 @@
+import constants from '../service/constants/constants';
+// import { GuildMember, Message } from 'discord.js';
+import fs from 'fs';
+import path from 'path';
+import { scoapEmbedState, botConvoState, voteRecordState } from '../service/scoap-squad/ScoapDatabase';
+
+const isInteger = (value: any): boolean => {
+	return /^\d+$/.test(value);
+};
+
+const ScoapUtils = {
+
+	validateTotalNumberOfRoles(message_content: string): boolean {
+		return (isInteger(message_content) && (parseInt(message_content) < 10) && (parseInt(message_content) >= 1));
+	},
+
+	validateTotalNumberOfPeoplePerRole(message_content: string): boolean {
+		return (isInteger(message_content) && (parseInt(message_content) < 1000) && (parseInt(message_content) >= 1));
+	},
+
+	validateSummary(summary: string): boolean {
+		const CREATE_SUMMARY_REGEX = /^[\w\s.!@#$%&,?']{1,4000}$/;
+		return (!(summary == null || !CREATE_SUMMARY_REGEX.test(summary)));
+	},
+
+	validateReward(message_content: string): boolean {
```

Comment:
This looks similar to the utils in bounty, can you reuse BountyUtils.validateReward?

Reply:
Yes, I reused the validation code from BountyUtils. The difference is they return booleans used to switch the message flow. Also the method in BountyUtils sends a DM which does not have the same format as the rest of the DM flow in scoap squad. I would prefer to keep it if that is ok.